bitnami/kafka/3.2/debian-11/rootfs/opt/bitnami/scripts/kafka-env.sh

```diff
@@ -95,6 +95,9 @@ export KAFKA_OPTS="${KAFKA_OPTS:-}"
 export KAFKA_CFG_SASL_ENABLED_MECHANISMS="${KAFKA_CFG_SASL_ENABLED_MECHANISMS:-PLAIN,SCRAM-SHA-256,SCRAM-SHA-512}"
 export KAFKA_KRAFT_CLUSTER_ID="${KAFKA_KRAFT_CLUSTER_ID:-}"
 export KAFKA_SKIP_KRAFT_STORAGE_INIT="${KAFKA_SKIP_KRAFT_STORAGE_INIT:-false}"
+export KAFKA_CFG_MAX_REQUEST_SIZE="${KAFKA_CFG_MAX_REQUEST_SIZE:-1048576}"
+export KAFKA_CFG_MESSAGE_MAX_BYTES="${KAFKA_CFG_MESSAGE_MAX_BYTES:-1048576}"
+export KAFKA_CFG_MAX_PARTITION_FETCH_BYTES="${KAFKA_CFG_MAX_PARTITION_FETCH_BYTES:-1048576}"
```

Comment:
Could you tell me why 1048576 is the default value?

Reply:
https://kafka.apache.org/documentation/#producerconfigs_max.request.size


Default: | 1048576